### ISSUE TITLE
Transaction Details > show dispute resolve status (won/lost/under review) & relevant action button in payment details card footer

### ISSUE THREAD

**Issue by shendy-a8c** [Automattic/woocommerce-payments#6927]
Part of #6883 

On the Transaction Details screen, we should show information about the status of a dispute that a merchant has challenged, accepted or ignored.

![Screenshot 2023-08-04 at 09 25 49](https://github.com/Automattic/woocommerce-payments/assets/73803630/e310c954-9e3b-423d-a133-b318809071af)

### Dispute status requirements

* Lost via challenge
	* `This dispute was lost on December 14, 2023. The $15 USD fee has been deducted from your account, and the disputed amount returned to the cardholder. Learn more about preventing disputes.`
	* Action button: **View dispute details** links to dispute challenge form (read-only)
	* Color: new grey (`#f2f4f5`)
* Accepted (`disputes.metadata.__closed_by_merchant`)
	* `This dispute was accepted and lost on June 16, 2023. The $15 USD fee has been deducted from your account, and the disputed amount returned to the cardholder. Learn more about preventing disputes.`
	* No button (handled in issue #7156)
	* Color: new grey (`#f2f4f5`)
* Lost due to non-response (no `dispute.metadata.__evidence_submitted_at`)
	* `This dispute was lost on December 14, 2023 due to non-response. The $15 USD fee has been deducted from your account, and the disputed amount returned to the cardholder. Learn more about preventing disputes.`
 	* No button (handled in issue #7156)
	* Color: new grey (`#f2f4f5`)
* Under review
	* `You submitted evidence for this dispute on June 16, 2023. The cardholder’s bank is reviewing the case, which can take 60 days or more. You will be alerted when they make their final decision. Learn more about the dispute process.`
	* Action button: **View submitted evidence** links to dispute challenge form (read-only)
	* Color: `$wp-blue-0`
* Won
	* `Good news! You won this dispute on December 14, 2023. The disputed amount and the dispute fee have been credited back to your account. Learn more about preventing disputes.`
	* Action button: **View dispute details** links to dispute challenge form (read-only)
	* Color: new grey (`#f2f4f5`)

#### Additional Details

* Design on Figma: RiEQmaKRI7u9USAI3lyZz0VX-fi-7047_436287
* Related issue for inquiries: #7191
* **View dispute details** buttons for `Lost: no response` and `Lost: accepted` disputes are no longer part of this PR – now handled in issue #7156.

**Comment by Jinksi** [comment]
> **Note**
> **View dispute details** buttons for `Lost: no response` and `Lost: accepted` disputes are no longer part of this PR – now handled in issue #7156.

**Comment by Jinksi** [comment]
I've moved the inquiry messages to #7191 to unblock this issue for disputes.

**Comment by haszari** [comment]
FYI I've updated the title for clarity – as I understand it, the purpose of the "footer"/message is to show the status (i.e. won, lost, draft response). 

One minor thing I'm confused about – why is `needs_response` out of scope for this issue? Is that because it's already implemented or is it a separate issue?

**Comment by haszari** [comment]
> One minor thing I'm confused about – why is needs_response out of scope for this issue? Is that because it's already implemented or is it a separate issue?

I get it now – this issue is focused on showing status of inactive/completed disputes, and an action button if relevant.

**Comment by Jinksi** [comment]
👍 I've also added "under review" to the title because this issue covers that status, which is sort of but not resolved.
